### DISCUSSION TITLE
Time Period

### DISCUSSION THREAD

**Discussion by jonathan2667** [farneet24/Company-wise-Leetcode#1] [General]
Hello, I have a question: last 6 months, last 1 year, etc, from when?
Is it up-to-date or made using a database from the past?
Thank you!

**Comment by farneet-singh** [comment]
Hello, I am using data from a repository, and from the commit history, it appears the data is 2-3 years old. A while back, I purchased a LeetCode premium subscription and recently took screenshots of some questions about two months ago. I've uploaded these screenshots to a drive, which you can access through the link I've included. I haven't used this data on the website as I could not organize it in a better format.

[LeetCode Recent Questions](https://drive.google.com/drive/folders/1Xq6JsvTYDQ85skybfqXR63gg54QoJkhv?usp=sharing)
